freqtrade/freqai/prediction_models/ReinforcementLearner.py

```diff
@@ -109,15 +108,15 @@ def calculate_reward(self, action: int) -> float:
             # reward agent for entering trades
             if (action == Actions.Long_enter.value
                     and self._position == Positions.Neutral):
-                self.custom_info[f"{Actions.Long_enter.name}"] += 1
+                self.custom_info[Actions.Long_enter.name] += 1
```

Comment:
Should we add a note in the documentation telling users how they can use `custom_info`?

Reply:
> Should we add a note in the documentation telling users how they can use `custom_info`?

How about renaming `custom_info` to something like `metrics`?

Reply:
> > Should we add a note in the documentation telling users how they can use `custom_info`?
> 
> How about renaming `custom_info` to something like `metrics`?

Renaming a dictionary is a good point, because the documentation defines custom_ variables as the recommended naming for user-defined variables, not built-in ones.

Reply:
Best would be `tensorboard_metrics` or something to indicate that the value will be available in tensorboard. Also, we already have a `metric_tracker` which tracks resource usage in freqai.